var/spack/repos/builtin/packages/py-pbr/package.py

```diff
@@ -29,8 +29,12 @@ class PyPbr(PythonPackage):
     """PBR is a library that injects some useful and sensible default
        behaviors into your setuptools run."""
     homepage = "https://pypi.python.org/pypi/pbr"
-    url      = "https://pypi.python.org/packages/source/p/pbr/pbr-1.8.1.tar.gz"
+    url      = "https://pypi.io/packages/source/p/pbr/pbr-1.10.0.tar.gz"
 
+    version('1.10.0', '8e4968c587268f030e38329feb9c8f17')
     version('1.8.1', 'c8f9285e1a4ca6f9654c529b158baa3a')
 
     depends_on('py-setuptools', type='build')
+    # Only needed for py<3.4, however when='^python@:3.4.2' syntax is broken
```

Comment:
We can't until the concretizer is fixed. I've ran into this problem before myself. In the meantime, it doesn't hurt to build it for a version of Python that already has it.